source/jobGeneration.js

```diff
@@ -39,6 +41,7 @@ const CONFIGURABLE_JOB_KEYS = [
 ];
 const DEFAULT_JOB_TYPE = "generic";
 const JOB_VALIDATION = {
+    archived: [a => typeof a === "boolean"],
```

Comment:
Maybe this branch needs a rebase when we merge the other one, so these don't appear?